cmake/scripts/run_test.sh

```diff
@@ -40,7 +40,7 @@ run(){
   rm -f failing_output
   rm -f output
 
-  ${RUN_COMMAND} > stdout
+  ${RUN_COMMAND} > stdout 2>&1
```

Comment:
What do you mean by "done"? So with this change stderr will no longer be reported if the test also generates "output"? That looks weird.